core/src/main/java/com/datastax/oss/driver/internal/core/adminrequest/AdminRequestHandler.java

```diff
@@ -117,7 +122,14 @@ protected AdminRequestHandler(
 
   public CompletionStage<ResultT> start() {
     LOG.debug("[{}] Executing {}", logPrefix, this);
-    channel.write(message, false, customPayload, this).addListener(this::onWriteComplete);
+    if (preAcquireId && !channel.preAcquireId()) {
```

Comment:
This is a bit confusing to me, from what I understand, the `preAcquireId` means that is should pre-acquire, and then `channel.preAcquireId()` is doing actual acquisition of stream id, right? If so, I propose to rename local field `preAcquireId` to `shouldPreAcquireId`, wdyt?

Reply:
Yeah I see what you mean, I will rename the boolean field.